pandas/_libs/parsers.pyx

```diff
@@ -71,9 +73,6 @@ cdef:
     float64_t NEGINF = -INF
 
 
-cdef extern from "errno.h":
```

Comment:
Pulling out errno ideally could have been a separate PR but I'm fine doing it here